takecare19/takecare19
# Add OG meta tags

### Testing instructions
* Check the correct tags are seen here https://opengraphcheck.com/
* Check when sharing the url i.e. twitter/slack you can see the OG image
